Tests/TuistLoaderTests/Models+ManifestMappers/TestingOptions+ManifestMapperTests.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import ProjectDescription
+import TSCBasic
+import TuistCore
+import TuistGraph
+import TuistSupport
+import XCTest
+
+@testable import TuistLoader
+@testable import TuistSupportTesting
+
+final class TestingOptionsManifestMapperTests: TuistUnitTestCase {
+    private typealias Manifest = ProjectDescription.Config.GenerationOptions.TestingOptions
```

Comment:
I find this a bit confusing - why don't we initialize the whole `Config`?

Reply:
I don't have any strong opinions here so happy to change, the main reason for this approach was to follow existing style, e.g. https://github.com/tuist/tuist/blob/main/Tests/TuistLoaderTests/Models%2BManifestMappers/CodeCoverageMode%2BManifestMapperTests.swift

I assume this is originally done this way since this is testing a function from `ProjectDescription.TestingOptions -> TuistGraph.TestingOptions` so creating the extra container of config wouldn't be necessary